java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java

```diff
@@ -57,4 +57,24 @@ CompletableFuture<Long> geoadd(
      * }</pre>
      */
     CompletableFuture<Long> geoadd(String key, Map<String, GeospatialData> membersToGeospatialData);
+
+    /**
+     * Returns the positions (longitude,latitude) of all the specified <code>members</code> of the
+     * geospatial index represented by the sorted set at <code>key</code>.
+     *
+     * @see <a href="https://valkey.io/commands/geopos">valkey.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param members The members for which to get the positions.
+     * @return A 2D <code>array</code> which represent positions (longitude and latitude)
+     *     corresponding to the given members. If a member does not exist, its position will be <code>
+     *     null</code>.
+     * @example
+     *     <pre>{@code
+     * client.geoadd("mySortedSet", Map.of("Palermo", new GeospatialData(13.361389, 38.115556), "Catania", new GeospatialData(15.087269, 37.502669)));
+     * Double[][] result = client.geopos("mySortedSet", new String[]{"Palermo", "Catania", "NonExisting"}).get();
+     * Double[][] expected = new Double[][]{{13.36138933897018433, 38.11555639549629859}, {15.08726745843887329, 37.50266842333162032}, null};
```

Comment:
a comment why these values are different than what was given, example: 
```
* // the coordinates are stored as a 52 bit geohash
```

Reply:
You can compare them with delta or just print the output as we do in couple other examples.